docs/releases_review/demo_v0.7.5.py

```
@@ -0,0 +1,309 @@
+async def demo_1_docker_hooks_system():
+    """Demo 1: Docker Hooks System - Real API calls with custom hooks"""
+    print_section(
+        "Demo 1: Docker Hooks System",
+        "Testing real Docker hooks with live API calls"
+    )
+
+    # Check Docker service availability
+    def check_docker_service():
+        try:
+            response = requests.get("http://localhost:11234/", timeout=3)
+            return response.status_code == 200
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)